### DISCUSSION TITLE
Advice if you have time

### DISCUSSION THREAD

**Discussion by AbhiMishra129** [official-stockfish/Stockfish#5380] [General]
Im 15 yr old started my journey with c++ language and working my way up to DSA to AI Engineer 

Can you guy tell me your experience about how should I continue my journey as Im  currently in grade 11

**Comment by FunnyStepBK** [comment]
Hi! Abhi I am too a newbie and just started with programming in cpp about 2 months ago, currently working on making a chess engine from scratch would you like to collaborate and work together.

**Reply by xu-shawn** [reply]
Many engine devs are active at the [official stockfish discord](https://discord.gg/GWDRS3kU6R).

**Reply by FunnyStepBK** [reply]
> Many engine devs are active at the [official stockfish discord](https://discord.gg/GWDRS3kU6R).

Oh Thanks!
